content/docs/rfcs/67/README.md

```diff
@@ -0,0 +1,200 @@
+---
+slug: 67
+title: 67/STATUS-WAKU2-USAGE
+name: Status Waku2 Usage
+status: raw
```

Comment:
Would this be `raw` or `draft`, since presumably there is a reference implementation?

Reply:
@jm-clius This is considered raw since all new specs should be raw.